package.json

```diff
@@ -71,7 +72,7 @@
     "wdio-mocha-framework": "0.3.1",
     "webdriverio": "4.0.9",
     "whatwg-fetch": "0.10.1",
-    "yuidoc-lucid-theme": "git+ssh://git@github.com:Shopify/yuidoc-lucid-theme.git#c9c62ba",
+    "yuidoc-lucid-theme": "git+ssh://git@github.com/Shopify/yuidoc-lucid-theme.git#c9c62ba",
```

Comment:
Did this have to change? If it did, ok, but it's weird. NPM has waffled on this format of URI a lot :/